spec/spec_helper.rb

```diff
@@ -17,6 +17,7 @@
   c.filter_sensitive_data('<UPS_LOGIN>') { ENV['UPS_LOGIN'] }
   c.filter_sensitive_data('<UPS_KEY>') { ENV['UPS_KEY'] }
   c.filter_sensitive_data('<UPS_PASSWORD>') { ENV['UPS_PASSWORD'] }
+  c.filter_sensitive_data('UPS_SHIPPER_NUMBER') { ENV['UPS_SHIPPER_NUMBER'] }
```

Comment:
How about using curly braces?

Reply:
Difficult with JSON.

Reply:
Ah, so JSON raises a ParseError if we use angle brackets?

Reply:
The commit messages talks about XML. Was denn nun?

Reply:
No, XML fails to parse XML with angle brackets.

Reply:
I don't think we need ANY brackets around theses strings actually. Can we move that discussion to another PR? It happened to surface here and brought me some very sad minutes (things passed locally with my `.env`, but not without it). But for now, this works, and replacing the replacement strings requires me to re-record all the cassettes. Meh. :)

Reply:
I tried 🙈 - but that doesn't work either, it convert's any text's encoding to unicode :D. Trying `%` now, which carries no meaning in either XML or JSON.